### ISSUE TITLE
Hebrew addon: all words are underlined in red

### ISSUE THREAD

**Issue by shoomkloom** [AnySoftKeyboard/AnySoftKeyboard#652]
In the latest update a small but annoying problem has come up when I use Hebrew: all the words are underlined in red as if misspelled... I sent an email with a screenshot.
Hope you can fix this soon  : -)

**Android OS version:**
6.0.1

**Device manufacturer and model:**
Nexus 6

**List of installed add-ons (like languages, or themes):**
Hebrew

**Comment by shoomkloom** [comment]
![screenshot_20160517-131149](https://cloud.githubusercontent.com/assets/5113600/15319309/e74adfd0-1c32-11e6-8a50-e4763a376be4.png)

**Comment by Omer26** [comment]
I have the same problem, and it's terrible! I can't use the keyboard anymore since it was updated today.

I have a OnePlus Two, Android 5.1.1

**Comment by menny** [comment]
This is something that started with the last update?
Did anything else was updated on you phone?

Also,can you check the spellchecker setting at the phone's settings?

**Comment by Omer26** [comment]
Yes, it started yesterday, I believe following the last update in the app.
Google keyboard was also updated lately, I think maybe two days ago - but
as I said, the google keyboard is working ok regarding the spell check.

The spell checker is the Google spell checker, and it is on. Is there any
way to keep it on only for English and not Hebrew? If so I will not care
about the problems in your keyboard regarding spelling, because I don't
need spell check in Hebrew.

Thank you,
Omer

**Comment by menny** [comment]
@shoomkloom @Omer26 please sign up to the beta program: 
https://play.google.com/apps/testing/com.menny.android.anysoftkeyboard

I'll provide a _possible_ fix shortly.

**Comment by menny** [comment]
Have you tried the latest beta?

**Comment by shoomkloom** [comment]
From a quick check just now, it seems to be solved.
Thanks for the quick response!

**Comment by shoomkloom** [comment]
And... it's back...
First time I used it it was ok, now I started telling someone that it was fixed and it's happening again...

**Comment by menny** [comment]
I noticed that I get the same behavior with Google's keyboard.
Can you confirm?

**Comment by menny** [comment]
Also, can you try the previous release: https://github.com/AnySoftKeyboard/AnySoftKeyboard/releases/download/1.7-r7/AnySoftKeyboard-release.apk
Does it work better?

**Comment by shoomkloom** [comment]
Sorry for the late reply...

I installed the version you posted here and the problem returned as before. 

The beta version I had before was behaving mostly OK except from time to time I got the red line under all or part of the words.
Still it was much better than the latest market version.
I'm sorry this is not so helpful, nothing as annoying as a sporadic bug...

I didn't see this happen with Google's keyboard, that's why I added a bug report.

**Comment by shoomkloom** [comment]
I played around with it again just now (the previous release you posted here) and the problem was back as I said.
I switched to Google's keyboard and the problem disappeared.
Then I switched back to Anysoft and now everything is working (mostly) ok... by that I mean that most of the words are not red underlined (one or two are, even though they are not misspelled).
I tried it in several different apps (e.g. WhatsApp, Google messenger) and they all behave more or less the same.

If you would like to send me a version with extra logging, etc. I would be happy to assist.

Cheers,
Oren
